metadata.json

```diff
@@ -25,6 +25,12 @@
     { "operatingsystem": "Scientific", "operatingsystemrelease": [ "6", "7" ] },
     { "operatingsystem": "Fedora",     "operatingsystemrelease": [ "19" ] },
     { "operatingsystem": "Debian",     "operatingsystemrelease": [ "6", "7" ] },
-    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] }
+    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] },
+    { "operatingsystem": "Windows",    "operatingsystemrelease": 
+      [ 
+        "Vista", "7", "8",
+        "2008", "2008 R2", "2012", "2012 R2"
+      ]
+    }
   ]
```

Comment:
@mmoll let's go with this if it works for you. Even though this is "just JSON" it does help when it's easy to read. I assume that's why the other OS's are on a single line. An alternative is to apply standard JSON formatting like puppet_blacksmith, Geppetto, and jsonlint.com do. That makes it look like this:
```json
"operatingsystem_support": [
        {
            "operatingsystem": "RedHat",
            "operatingsystemrelease": [
                "6",
                "7"
            ]
        },
        {
            "operatingsystem": "CentOS",
            "operatingsystemrelease": [
                "6",
                "7"
            ]
        },
        {
            "operatingsystem": "Scientific",
            "operatingsystemrelease": [
                "6",
                "7"
            ]
        },
        {
            "operatingsystem": "Fedora",
            "operatingsystemrelease": [
                "19"
            ]
        },
        {
            "operatingsystem": "Debian",
            "operatingsystemrelease": [
                "6",
                "7"
            ]
        },
        {
            "operatingsystem": "Ubuntu",
            "operatingsystemrelease": [
                "12.04",
                "14.04"
            ]
        },
        {
            "operatingsystem": "Windows",
            "operatingsystemrelease": [
                "Vista",
                "7",
                "8",
                "2008",
                "2008 R2",
                "2012",
                "2012 R2"
            ]
        }
    ]
```